cmd/cloudStorage/main.go

```diff
@@ -1,23 +1,28 @@
 package main
 
-//go:generate sh -c "mkdir -p ../../gen/storage && swagger generate server -A storage -t ../../gen/storage -f ../../docs/api/storage.yml --exclude-main --principal string"
+//go:generate sh -c "mkdir -p ../../gen/storage && swagger generate server -A storage -t ../../gen/storage -f ../../docs/api/storage.yml  --with-context  --exclude-main --principal string"
```

Comment:
`--with-context` is deprecated. Golang now includes context in requests, so you can use `params.HTTPRequest.Context()` in your handlers.

Reply:
uh, thanks.